ADR/0011-roles-and-permissions.md

```diff
@@ -0,0 +1,81 @@
+# 11. Roles and Permissions for Stonesoup
+
+Date: 2023-01-10
+
+## Status
+
+Accepted
+
+## Context
+Stonesoup is using Kubernetes as the control plane for managing its resources. We require a system for managing user roles and permissions in this context. We have defined the following roles for our project: "Contributor", "Maintainer", and "Owner". We need to map these roles to specific permissions in the Kubernetes RBAC system, in terms of API groups, verbs, and resources.
+
+## Decision
+
+We will use the built-in Kubernetes RBAC system for Stonesoup's role and permissions management, and map the following roles to specific permissions, as described in the table below:
+
+### Roles
+**Contributor:** Members who interact with the workspace mostly through pull requests.
+**Maintainer:** Members who manage the workspace without access to sensitive or destructive actions.
+**Owner:** Members who have full access to the workspace including sensitive and destruction actions.
```

Comment:
So, now after the today's discussion we need to rename `Owner` to `Admin`, right?